playground/connect_db.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import Column, Integer, String
+from sqlalchemy.orm import sessionmaker
+
+"""[reference] https://docs.sqlalchemy.org/en/13/orm/tutorial.html"""
+
```

Comment:
the code itself is quite straightforward and has no problem,
but could you consider separating this part as a module? e.g.:

```python
class DBConfig(object):
    def __init__(self):
       self.session_maker = None

    def setup_db_connection(self, sqlalchemy_uri, echo=False):
        engine = create_engine(sqlalchemy_url, echo=echo)
        self.session_maker = sessionmaker(bind=engine)
```

so that we can utilize this code from other places like
when we init 
```python
config = DBConfig()
config.setup_db_connection(uri="<our uri>")
session_maker = config.session_maker
```

and utilize
```python
def get_test_table(role):
    session = session_maker()
    return session.query(TESTTABLE).filter_by(age=31).all()
```

Reply:
Yeah!